### DISCUSSION TITLE
[cpp2sky]The cpp server trace is always broken

### DISCUSSION THREAD

**Discussion by wuwen5** [apache/skywalking#9576] [General]
I tested the Cpp client to access the Java service, and everything was OK, But from the Cpp client to the Cpp service was always broken.

![image](https://user-images.githubusercontent.com/5037807/188817812-194555fd-dc1a-4fa9-ac4c-674482f9baa1.png)

![image](https://user-images.githubusercontent.com/5037807/188821307-cd73a81f-78e6-44c8-be07-09f8265f13d0.png)

![image](https://user-images.githubusercontent.com/5037807/188818343-0927b997-242d-4331-b2af-520cefe5d635.png)

```
echo NDc3NTY5NDItMzA0My00MTQ5LTlhNTUtMzY2ZTYyNDc1NTc0 |base64 -d
47756942-3043-4149-9a55-366e62475574%
```

**Comment by wu-sheng** [comment]
cpp2sky is not an Apache project. Please move to the project issue list.
